src/FSharpVSPowerTools.Logic/VSUtils.fs

```diff
@@ -58,16 +53,16 @@ let isPhysicalFileKind (kind: string) =
     kind.Equals(EnvDTE.Constants.vsProjectItemKindPhysicalFile, StringComparison.OrdinalIgnoreCase)
 
 let isPhysicalFileOrFolderKind kind =
-    kind <> null && (isPhysicalFolderKind kind) || (isPhysicalFileKind kind)
+    isNotNull kind && (isPhysicalFolderKind kind) || (isPhysicalFileKind kind)
```

Comment:
I think `isNotNull` is as clear as `<> null`. I try to be consistent by using one approach and not to have to worry about whether the check will be in a tight loop.